workshops/views.py

```diff
@@ -161,6 +177,10 @@ class DeleteViewContext(DeleteView):
     """
     Class-based view for deleting objects that extends default template context
     by adding proper page title.
+
+    GET requests are redirected to appropriate update views.
```

Comment:
"GET to /workshops/object/delete → redirect to /workshops/object/", is that what this is doing?

Why not throw 405 Method Not Allowed…? It's safer than redirection + error message "sorry but you must have mistakenly entered that other url, well hell no, you should click this red button on the bottom instead".